### ISSUE TITLE
Show a11y errors in the form preview

### ISSUE THREAD

**Issue by pinussilvestrus** [camunda/form-playground#75]
__Is your feature request related to a problem? Please describe.__

We want our users to build accessible forms. We could use the form preview to display a11y errors directly (e.g. a label is missing, an alternative text is missing, cf. [axe `cat.forms` category](https://dequeuniversity.com/rules/axe/4.5)). With increasing form components, this could become more important as forms get more complex.

We are heavily using `axe` under the hood and we can consider using the same foundations to display errors visually.

__Describe the solution you'd like__

Show a11y errors on the form preview.

As an example, the axe dev tools show a lot of details around such detected errors.

![image](https://github.com/camunda/form-playground/assets/9433996/254d79b9-8ead-418e-a4d2-b66c52a4e9a4)

We could consider showing such information via a tooltip next to the respective form field.

__Describe alternatives you've considered__

Promote using existing dev tools and keep this functionality out of the form playground.

__Additional context__

/

**Comment by pinussilvestrus** [comment]
A spike was made via https://github.com/camunda/form-playground/pull/76

This only shows how to use axe do generate such errors and provide a link for "more", e.g. how to fix the issue.

I could imagine keeping this out of form-playground completely and implement this as part of https://github.com/camunda/form-linting (e.g. with showing the errors on the canvas as well, as we do with [BPMN linting](https://github.com/camunda/linting))

![Kapture 2023-06-02 at 13 29 40](https://github.com/camunda/form-playground/assets/9433996/a209f8c1-f9a2-4a58-9998-19d48fec8e20)
